src/frontend/src/components/public/PIASideNav/pia-form-sideNav-pages.ts

```diff
@@ -4,13 +4,125 @@
  * till the complete feature is implemented we can control what is disaplayed in the side nav
  */
 
+import { routes } from '../../../constant/routes';
+import { IPiaForm } from '../../../types/interfaces/pia-form.interface';
+import { buildDynamicPath } from '../../../utils/path';
 import { INavbarItem } from '../../common/Navbar/interfaces';
 
-export const piaFormSideNavPages: INavbarItem[] = [
-  {
-    id: 1,
-    label: 'PIA Intake',
-    link: '/pia/:id',
-    enable: true,
-  },
-];
+export const piaFormSideNavPages = (
+  pia: IPiaForm,
+  isEditMode = false,
+  isNewForm = false,
+): INavbarItem[] => {
+  // This will change once Next Steps tab is implemented
+  const showPostIntakeTabs =
+    !!pia?.hasAddedPiToDataElements && !!pia?.submittedAt;
+
+  const intakeLink = isNewForm
+    ? routes.PIA_NEW
+    : isEditMode
+    ? routes.PIA_INTAKE_EDIT
+    : routes.PIA_INTAKE_VIEW;
+
+  return [
+    {
+      id: 1,
```

Comment:
having id here is not useful. If we swap a nav then all the id changes. This can be dynamically generated.

Reply:
What react has done is perfect valid key={number.toString()} value={number} />
They take the value that should be uniquely identified in our case it is the name of the list and use that as a key. 

id to me makes the app very fragile.